assets/js/tx-sender.js

```diff
@@ -239,6 +239,17 @@ const set_up_tx_sender = async function() {
     }
     $("#pp_progress .progress-bar").width("20%")
 
+    // Wait for the address's funding to be validated so we don't get the wrong
+    // starting sequence number.
+    while (true) {
+      try {
+        await new Promise(resolve => setTimeout(resolve, 1000))
+        await api.request("account_info", {account: pp_issuer_address,
+          ledger_index: "validated"})
+        break
+      } catch(e) {}
+    }
```

Comment:
I ran into https://github.com/ripple/rippled/issues/3795 while developing the interactive tutorials, and realized that it's also the reason that the Transaction Sender tool's partial payments setup stopped working. (It was written before Deletable Accounts made this a thing.) Fortunately, the same fix I came up with for the interactive tutorials also works for the transaction sender.